src/test/java/net/coru/kloadgen/extractor/extractors/JsonExtractorTest.java

```diff
@@ -0,0 +1,294 @@
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import io.confluent.kafka.schemaregistry.json.JsonSchema;
+import net.coru.kloadgen.exception.KLoadGenException;
+import net.coru.kloadgen.model.ConstraintTypeEnum;
+import net.coru.kloadgen.model.FieldValueMapping;
+import net.coru.kloadgen.testutil.FileHelper;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class JsonExtractorTest {
+
+  private final FileHelper fileHelper = new FileHelper();
+
+  private final JsonExtractor jsonExtractor = new JsonExtractor();
+
+  @Test
+  void testBasic() throws Exception {
+    String testFile = fileHelper.getContent("/jsonschema/basic.jcs");
+
+    Map<ConstraintTypeEnum, String> constraints = new HashMap<>();
+    constraints.put(MINIMUM_VALUE, "0");
+    constraints.put(MAXIMUM_VALUE, "0");
+
+    List<FieldValueMapping> fieldValueMappingList = jsonExtractor.processSchema(new JsonSchema(testFile).toJsonNode());
+
+    assertThat(fieldValueMappingList)
+        .hasSize(3)
+        .containsExactlyInAnyOrder(
+            new FieldValueMapping("firstName", "string", 0, "", constraints, false, false),
+            new FieldValueMapping("lastName", "string", 0, "", constraints, true, false),
+            new FieldValueMapping("age", "number", 0, "", true, false)
+        );
+  }
+
+  @Test
+  void testBasicArray() throws Exception {
+    String testFile = fileHelper.getContent("/jsonschema/basic-array.jcs");
+
+    Map<ConstraintTypeEnum, String> constraints = new HashMap<>();
+    constraints.put(MINIMUM_VALUE, "0");
+    constraints.put(MAXIMUM_VALUE, "0");
+
+    List<FieldValueMapping> fieldValueMappingList = jsonExtractor.processSchema(new JsonSchema(testFile).toJsonNode());
+
+    assertThat(fieldValueMappingList)
+        .hasSize(3)
+        .containsExactlyInAnyOrder(
+            new FieldValueMapping("fruits[]", "string-array", 0, "", true, false),
+            new FieldValueMapping("vegetables[].veggieName", "string", 0, "", constraints, true, true),
+            new FieldValueMapping("vegetables[].veggieLike", "boolean", 0, "",true, true)
+        );
+  }
+
+  @Test
+  void testBasicNumber() throws Exception {
+    String testFile = fileHelper.getContent("/jsonschema/basic-number.jcs");
+
+    List<FieldValueMapping> fieldValueMappingList = jsonExtractor.processSchema(new JsonSchema(testFile).toJsonNode());
+
+    assertThat(fieldValueMappingList)
+        .hasSize(2)
+        .containsExactlyInAnyOrder(
+            new FieldValueMapping("latitude", "number", 0, "", new HashMap<ConstraintTypeEnum, String>() {{
+              put(MINIMUM_VALUE, "-90");
+              put(MAXIMUM_VALUE, "90");
+              put(EXCLUDED_MINIMUM_VALUE, "0");
+              put(EXCLUDED_MAXIMUM_VALUE, "0");
+              put(MULTIPLE_OF, "0");
+            }}, true, false),
```

Comment:
Instead of creating a new map inside another method call, we recomend using `Map.of(...)` to improve the readability and to keep the code as understandable as possible. 
That must be changed before the PR could be approved.